21120558/meogroup-backend
# Consolidated Pull Request Title: Update repository with changes including deletion and addition of files, .gitignore modification, and creation of new files

## Title
Consolidated Changes Pull Request

## Description
This pull request consolidates multiple changes including the addition of new files, deletion of existing files, modifications to the .gitignore file, and creation of new files with content.

## Changes
- Deleted "hehe.txt"
- Added "hehe.txt"
- Deleted "haha.txt"
- Added entry for ".ezcommit" in .gitignore file
- Created "haha.txt"
- Created "test2.txt" with 6 lines of text

## Testing
To test the changes, follow these steps:
1. Check that "hehe.txt" has been deleted
2. Verify the addition of "hehe.txt"
3. Confirm the deletion of "haha.txt"
4. Ensure the .gitignore file includes ".ezcommit"
5. Check the contents of "haha.txt"
6. Open "test2.txt" and verify the 6 lines of text

## Related Issues
None

## Checklist
- [ ] Changes reviewed
- [ ] All checks pass
